docs/3.x.x/en/plugin-development/utils.md

```diff
@@ -2,6 +2,63 @@
 
 Strapi provides helpers so you don't have to develop again and again the same generic functions.
 
+## Auth
+
+`auth.js` lets you get, set and delete data in either the browser's `localStorage` or `sessionStorage`.
+
+### Methods
+
+| Name | Description |
+| ---- | ––––––––––– |
+| clear(key) | Remove the data in either `localStorage` or `sessionStorage` |
+| clearAppStorage() | Remove all data from both storage |
+| clearToken() | Remove the user's `jwt Token` in the appropriate browser's storage |
+| clearUserInfo() | Remove the user's info from storage |
+| get(key) | Get the item in the browser's storage |
+| getToken() | Get the user's `jwtToken` |
+| getUserInfo() | Get the user's infos |
+| set(value, key) | Set an item in the `sessionStorage`. If `true` is passed a 3rd parameter it sets the value in the `localStorage` |
```

Comment:
Wouldn't it be better to move `get` and `set` functions in a util dedicated to storage (eg. `utils/storage`).

Reply:
Add `isLocalStorage` as parameter in this function.

Reply:
You're right but it's not related  with this PR